app/styles/ui/_diff.scss

```diff
@@ -286,17 +286,36 @@
   }
 }
 
-.image-diff-two-up {
+.image-diff-container {
   display: flex;
   flex: 1;
   justify-content: center;
   align-items: center;
+  flex-direction: column;
 
-  .image-diff-previous {
-    margin-right: 7.5px;
+  .image-diff-two-up {
+    display: flex;
+    align-items: center;
+
+    .image-diff-previous {
+      margin-right: 7.5px;
```

Comment:
@donokuda is this a good place to use `var(--spacing)` since the difference in values is only 2.5px?

Reply:
Thanks, good idea